src/quantum/eng/Generate-Client.ps1

```diff
@@ -0,0 +1,100 @@
+# --------------------------------------------------------------------------------------------
```

Comment:
Comment only: I'm glad that this got added to the repo. While it's not a part of this PR, I'd recommend a follow up change adding a Readme.md in this directory explaining how to use it as well as the other script.

Reply:
Yes, a Readme.md is a good idea.  I planned to write a wiki page about this procedure, but a Readme.md would be easier to find.